### ISSUE TITLE
Unhandled exception: [Enter a title]

### ISSUE THREAD

**Issue by DonCapiche** [remyroy/CDDA-Game-Launcher#31]
* Description: [Enter what you did and what happened]
* Version: 0.7.5
* Type: `<class 'OSError'>`
* Value: [WinError 145] The directory is not empty: 'C:\\Users\\DONCAP~1\\AppData\\Local\\Temp\\CDDA Game Launcher\\newbuild-091ed992'
* Traceback:
```
  File "cddagl\ui.py", line 1701, in timeout
  File "shutil.py", line 482, in rmtree
  File "shutil.py", line 381, in _rmtree_unsafe
  File "cddagl\ui.py", line 100, in remove_readonly

```

**Comment by remyroy** [comment]
This error is a little weird. In the update process, the new archived is downloaded in a temporary directory `C:\Users\DONCAP~1\AppData\Local\Temp\CDDA Game Launcher\newbuild-091ed992` in your case and it is removed afterwards. For some reason, your directory was not removed. There might be another process interfering with that. I'll see if I can add more information and the possibility to retry deleting that temporary directory.

**Comment by remyroy** [comment]
This should be fixed with a43708e7c6f2c7408b14cca02f546268f10858f3
